src/BloomExe/Book/TranslationGroupManager.cs

```diff
@@ -245,7 +245,7 @@ private static void MakeElementWithLanguageForOneGroup(XmlElement groupElement,
 		private static void StripOutText(XmlNode element)
 		{
 			var listToRemove = new List<XmlNode>();
-			foreach (XmlNode node in element.SelectNodes("descendant-or-self::*[(self::p or self::br) and not(contains(@class,'bloom-cloneToOtherLanguages'))]"))
+			foreach (XmlNode node in element.SelectNodes("descendant-or-self::*[(self::p or self::br or self::u or self::b or self::i) and not(contains(@class,'bloom-cloneToOtherLanguages'))]"))
```

Comment:
I don't think this code guarantees not to remove a b/i/u element which has children that might be images or marked clone to other languages. But the original didn't check this for <p> either. So I'm assuming the plan is to mark any such element with the special class.